R/genSlabLabels.R

```diff
@@ -1,54 +1,70 @@
-## TODO: documentation / generalization
-# note source data must be normalized via dice() first, e.g. all share the same number of horizons
+# does not assume logical horizons (may overlap, resulting in different # of slices per profile)
+.genSlabLabels2 <- function(spc, diced, slab.structure = 1) {
```

Comment:
Some commentary + pretty formatting would help me understand what is going on here.

Reply:
I added documentation and some comments. I am not sure if we want to export this function if this is the one we are going to use internally? I found that some of the assumptions in the old genSlabLabels didnt generalize as well to the less logical inputs, so wanted to create something that did the same thing / had apprx same output, but since inputs are different I created new function/definition